client-ui/src/app/components/lineage-overview/lineage-overview-details/lineage-overview-details.component.html

```diff
@@ -36,10 +36,23 @@ <h3 class="text-center pb-3 mt-3"> Execution Event </h3>
 
     <div class="divider"></div>
 
+    <!-- EXECUTION PLAN DETAILS -->
+    <div *ngIf="executedLogicalPlan$ | async as executedLogicalPlan" class="exec-plan-details">
+      <div ngbDropdown class="d-inline-block toolbar">
```

Comment:
It doesn't feel right to have a dropdown for one item, but the whole button might not fit into the same line. Maybe new line with button would be better?

Reply:
I tried... it sucks one way or another :( With a dropdown at least it's not that bulky, plus we have a decent place to add more actions if we need. 
I'll add a hint on the button to tell the user that the same action can be achieved by a double click on a node. So there will be less chance that a user would want to use it as a primary navigation control.

Reply:
OK, lets do it this way for now, but I think it would be good to revisit this later

Reply:
We're gonna have a new UI with (hopefully) better navigation :)